CHANGELOG.rst

```diff
@@ -40,6 +43,9 @@ Changed
 -------
 - new event broker class: ``SQLProducer``. This event broker is now used when running locally with
   Rasa X
+
+Removed
```

Comment:
I guess there was a merge conflict. `Removed` should not be here. Can you please go over the file again?